ERCS/erc-7602.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 7602
+title: Ethereum Mixing Transaction Supervision
+description: Monitoring and regulating mixing transactions on the Ethereum blockchain.
```

Comment:
Would be nice to expand the description to include a definition of "mixing transaction".